src/pybind/mgr/dashboard/controllers/mgr_modules.py

```diff
@@ -1,18 +1,42 @@
 # -*- coding: utf-8 -*-
 from __future__ import absolute_import
 
-from . import ApiController, RESTController
+from . import ApiController, RESTController, ControllerDoc, EndpointDoc
 from .. import mgr
 from ..security import Scope
 from ..services.ceph_service import CephService
 from ..services.exception import handle_send_command_error
 from ..tools import find_object_in_list, str_to_bool
 
+MGR_MODULE_SCHEMA = ([{
+    "name": (str, "Module Name"),
+    "enabled": (bool, "Is Module Enabled"),
+    "always_on": (bool, ""),
+    "options": ({
+        "Option_name": ({
+            "name": (str, "Option Name"),
+            "type": (str, "Module type"),
+            "level": (str, "Level"),
+            "flags": (int, ""),
+            "default_value": (int, ""),
+            "min": (str, ""),
+            "max": (str, ""),
+            "enum_allowed": ([str], ""),
+            "desc": (str, "Description"),
+            "long_desc": (str, "Long description"),
+            "tags": ([str], ""),
+            "see_also": ([str], "")
```

Comment:
It's similar to the config options, e.g.:

```suggestion
            "name": (str, "Name of the option"),
            "type": (str, "Type of the option"),
            "level": (str, "Option level"),
            "flags": (int, "List of flags associated"),
            "default_value": (int, "Default value for the option"),
            "min": (str, "Minimum value"),
            "max": (str, "Maximum value"),
            "enum_allowed": ([str], ""),
            "desc": (str, "Description of the option"),
            "long_desc": (str, "Elaborated description"),
            "tags": ([str], "Tags associated with the option"),
            "see_also": ([str], "Related options")
```